src/app/screens/Discover/websites.json

```diff
@@ -153,6 +153,12 @@
         "subtitle": "The blue bird experience for Nostr",
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/yosup_thumbnail.svg",
         "url": "https://yosup.app/"
+      },
+      {
+        "title": "Primal",
+        "subtitle": "Nostr highlights by Primal",
```

Comment:
This should be changed or left out.  
The current subtitle is a label from their UI:
<img width="840" alt="image" src="https://user-images.githubusercontent.com/1016218/225620117-040454b0-a1d5-41cb-91c8-9c15e55e9a39.png">

Reply:
Changes updates

Reply:
Okay, i am updating the subtitle